### PR TITLE
Fixed Emoji issue

### LINKED ISSUE #3: Emoji
Replace Unicode emoji characters in messages with the emoji name (like `:thumbsup:`) before sending, or maybe something like `:)` instead of `:smile:` etc for faces.

### PR DESCRIPTION
**Pull request by BTELNYY**
Fixed with the help or Earthoriod

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -255,7 +256,10 @@ client.on('message', async message => {
 			const role = message.member.roles.highest.name;
 			const name = message.member.displayName;
 			// let content = yourls.conditionalReplace(message.cleanContent, dependencies);
-			let content = message.cleanContent;
+			let content = emoji
+				.unemojify(content.cleanContent) // replace standard/unicode emojis
```
Comment (eartharoid) [inline index.js]: Should be `message.cleanContent`, not `content.cleanContent`.

**Review by eartharoid — CHANGES_REQUESTED** [review]
Good work *(except you forgot to remove the old function)*, but there's 1 problem that needs to be resolved.

File: index.js (hunk 1 above)
Comment (BTELNYY) [reply index.js]: oops

**Review by eartharoid — APPROVED** [review]
Thank you @BTELNYY.

**Pull request merged**